requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
-redis>=5.0.0
 pyyaml
 coloredlogs
-pydantic<2.0.0
+redis>=5.0.0
+pydantic==2.5.3
```

Comment:
may think about relaxing this a bit. Pinning to a patch version for this popular of a library is bound to cause some issues. more on this in the top level comment

Reply:
Will do. Good call